### ISSUE TITLE
Missing form attributes (entype/encoding) in front end forms

### ISSUE THREAD

**Issue by funkolector** [WebDevStudios/CMB2#736]
Hi there, and thanks for your great work :)

I've found a behaviour that I cannot explain !
I'm using 2 front end forms, both using a file type field (see example below)
The first is printing the enctype and enconding attributes on the form, but not the second, and this situation blocks the FILE submission...

Any idea?

### Expected Behavior:
1st form (member_account) :
`<form class="cmb-form" method="post" id="member_account" enctype="multipart/form-data" encoding="multipart/form-data">`

2nd form (brand_page_content_form) :
`<form class="cmb-form" method="post" id="brand_page_content_form" enctype="multipart/form-data" encoding="multipart/form-data">`

### Actual Behavior:
1st form (member_account) :
`<form class="cmb-form" method="post" id="member_account" enctype="multipart/form-data" encoding="multipart/form-data">`

2nd form (brand_page_content_form) :
**`<form method="post">`**

### CMB2 Field Registration Code:

```php
add_action( 'cmb2_init', array($forms, 'cmb2_member_account_metaboxes' ));
add_action( 'cmb2_init', array($forms, 'cmb2_brand_page_metaboxes' ));

class *_Forms {

	function cmb2_member_account_metaboxes() {

		$cmb = new_cmb2_box( array(
			'id'            => 'member_account',
			'object_types'	=> array( 'user' ), 
			'hookup'		=> false, 
			'save_fields'	=> false,
			'context'       => 'normal',
			'priority'      => 'high',
			'show_names'    => true, 
		) );

		$cmb->add_field( array(
			'name'		=> __( 'Photo'  ),
			'id'		=> 'profile_photo',
			'type'		=> 'text',
			'attributes'=> array(
				'type'	=> 'file', 
			),
		) );

	}
	function cmb2_brand_page_metaboxes() {

		$cmb = new_cmb2_box( array(
			'id'            => 'brand_page_content_form',
			'object_types'	=> array( 'ex1', 'ex2'   ), 
			'hookup'		=> false, 
			'save_fields'	=> false,
			'context'       => 'normal',
			'priority'      => 'high',
			'show_names'    => true,

		) );


		$cmb->add_field( array(
			'name'       => 'Photo2'
			'id'         => 'profile_photo2',
			'type'       => 'text',
			'attributes' => array(
				'type' => 'file', 
			),

	}
}

**Comment by tw2113** [comment]
@funkolector Did you figure out the issue with this one? If yes, would you be willing to provide the solution you found, in case someone else stumbles onto this.

**Comment by funkolector** [comment]
Hi @tw2113!

Yes I closed the issue right away when I better looked at my code...
I found this "trouble" was due to an obvious, but unexpected :), mistake of mine: I had left another form wrapping the cmb_form... which caused the problem...

It was actually the first time I had a trouble with CMB2, and this confirmed the quality of your framework.
Thanks again, it is a real pleasure to use and a real productivity tool for my projects.

Hope the best to you and the team.
